election-ui-swift/Modules/ElectionResults/Sources/ElectionResults/Public/ResultsService.swift

```diff
@@ -1,12 +1,17 @@
 
 import Foundation
 
+/// Error descriptor used when fetching election results
+public enum ResultsServiceError: Error {
+    /// Represents an error decoding the JSON data retrieved by the service. The JSON data may be malformed, or doesn't match the expected structure.
+    case invalidJSON
```

Comment:
This is slightly nit-picky. But the ResultsService shouldn't know it's dealing with JSON. That is the responsibility of the Repository. Maybe something more generic. Like invalidResponse.

Reply:
Cool I'll fix in the iOS tech assessment and bring it in with the next set of changes